src/EditProvider.ts

```diff
@@ -58,7 +59,7 @@ export class EditProvider
     const fileExtension = this.fileExtensionForDocument(document);
     const filePath = document.fileName;
     const projectPath = vscode.workspace.rootPath;
-    return EditProvider.beautifyOptions(filePath).then(beautifyOptions => {
+    return EditProvider.beautifyOptions(projectPath).then(beautifyOptions => {
```

Comment:
What is the reason for this change? Previously this allowed Unibeautify to lookup the configuration file closest to the `filePath`.

Reply:
because in version 5 of `cosmiconfig` it will `search` inside a directory or `load` a config file. I tried `filePath`but got an error every time, because `cosmiconfig` was trying to load the changed file as a config. 

```
notificationsAlerts.ts:42 We could not find your configuration file: 
/Users/marvinheilemann/Documents/Projects/_extensions/unibeautify-vscode/testProject/typescript/test.ts. 
Please correct your path, otherwise the plugin will not work!
```
error from line 159

Anyway we could leave it blank, because this is the default passed string on line 103